File: Submissions/T19/H3/h3.py
def faktorial(n): return 1 if n <= 1 else n * faktorial(n - 1)
def pangkat(a, b): return 1 if b == 0 else a * pangkat(a, b - 1)

n = int(input("Masukkan n (faktorial) : "))
a, b = map(int, input("Masukkan a dan b : ").split())
print(f"{n}! = {faktorial(n)}")
print(f"{a}^{b} = {pangkat(a, b)}")

File: Submissions/T19/H3/h3.c
#include <stdio.h>
long long faktorial(int n) { return n <= 1 ? 1 : n * faktorial(n - 1); }
long long pangkat(int a, int b) { return b == 0 ? 1 : a * pangkat(a, b - 1); }
int main() {
    int n, a, b;
    printf("Masukkan n (faktorial) : "); scanf("%d", &n);
    printf("Masukkan a dan b : "); scanf("%d %d", &a, &b);
    printf("%d! = %lld\n", n, faktorial(n));
    printf("%d^%d = %lld\n", a, b, pangkat(a, b));
}
